src/frontend/components/Felleskomponenter/Barnetrygdperiode/spørsmål.ts

```diff
@@ -0,0 +1,21 @@
+export enum BarnetrygdperiodeSpørsmålId {
+    barnetrygdsperiodeEøs = 'barnetrygdsperiode-eøs',
+    mottarEøsBarnetrygdNå = 'mottar-barnetrygd-nå',
+    barnetrygdsland = 'barnetrygdsland',
+    fraDatoBarnetrygdperiode = 'fra-dato-barnetrygdperiode',
+    tilDatoBarnetrygdperiode = 'til-dato-barnetrygdperiode',
+    månedligBeløp = 'månedlig-beløp',
+}
+
+export const barnetrygdperiodeSpørsmålSpråkId = (
+    tilbakeITid = false
```

Comment:
Tenker også kanskje at akkurat her er det litt skummelt å presette dette parameteret, for jeg tror kanskje vi i alle tilfeller ønsker å sende det inn? 🤔